src/audit-license-headers.js

```diff
@@ -44,11 +46,27 @@ var COMMON_RAT_EXCLUDES = [
     '*.xcworkspacedata',
     '*.xccheckout',
     '*.xcscheme',
-    ];
+];
+
+var RAT_IGNORE_PATH          = '.ratignore';
+var RATIGNORE_COMMENT_PREFIX = '#';
+
+var RAT_NAME = 'apache-rat-0.10';
+var RAT_URL  = 'https://dist.apache.org/repos/dist/release/creadur/apache-rat-0.10/apache-rat-0.10-bin.tar.gz';
+
+function startsWith(string, prefix) {
+    return string.lastIndexOf(prefix, 0) === 0;
```

Comment:
You're right, `indexOf` is much more clear. `startsWith("boomboom","boom");` actually is `true`, because it's doing `lastIndexOf` starting at **0**, but it's clearly confusing. Fixed.